source/object-keys.ts

```diff
@@ -0,0 +1,20 @@
+/**
+A typed implementation of `Object.keys()`.
+
+This is useful since `Object.keys()` returns an array of strings. This function returns an array of the type keys of the given object.
```

Comment:
Can you link to the TypeScript issue about improving the Object.keys type.

Reply:
- Closed in issue: https://github.com/microsoft/TypeScript/issues/45390
- Reasoning: https://stackoverflow.com/questions/55012174/why-doesnt-object-keys-return-a-keyof-type-in-typescript